content/spin/cli-reference.md

```diff
@@ -410,6 +508,31 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
+{{ startTab "v1.3.0"}}
+
+<!-- @selectiveCpy -->
+
+```console
```

Comment:
All of this going to depend on what version of the cloud plugin they have installed.  It might be better to link to https://developer.fermyon.com/cloud/cloud-plugin which covers the cloud commands with appropriate versioning.  But I am not sure what the marketing position is, if we want to include cloud commands in the Spin docs.